examples/styled-jsx-with-csp/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "styled-jsx-with-csp",
+  "version": "1.0.0",
+  "private": true,
+  "author": {
+    "name": "Karthikeyan Ranasthala",
+    "email": "hello@karthikeyan.sh"
+  },
```

Comment:
```suggestion
```

While we appreciate your contribution, we don't want your email to be in every person's repo that clones this example 😄 Your contribution will still be tracked through GitHub!